src/pages/timer.js

```diff
@@ -0,0 +1,107 @@
+import {useState, useEffect} from 'react';
+
+import CookingTimer from '../components/CookingTimer';
+import {StyledInput, StyledInputTime} from '../components/styled/StyledInput';
+
+function Timer() {
+	const [timerDays, setTimerDays] = useState();
+	const [timerHours, setTimerHours] = useState();
+	const [timerMinutes, setTimerMinutes] = useState();
+	const [timerSeconds, setTimerSeconds] = useState();
+	const [userInputDays, setUserInputDays] = useState(0);
+	const [userInputHours, setUserInputHours] = useState(0);
+	const [userInputMinutes, setUserInputMinutes] = useState(0);
+	const [userInputSeconds, setUserInputSeconds] = useState(0);
+
+	//Ich möchte dass der User z.B. eine Stundenanzahl eintragen kann, und dann der Countdown von dem Wert (z.B. 8h30min) runterzählt evtl statt 'October,8...' einfach Date.now() + (UserInput) = deadline ? und danach bei getTime deadline - date.now
```

Comment:
```suggestion


```

Reply:
Yea you are right, i should remove all unrelevant comment and codes.